test/cluster/mv/tablets/test_mv_tablets.py

```diff
@@ -327,7 +327,7 @@ async def test_mv_tablet_split(manager: ManagerClient):
         '--target-tablet-size-in-bytes', '1024',
     ]
     servers = [await manager.server_add(config={
-        'error_injections_at_startup': ['short_tablet_stats_refresh_interval']
+        'tablet_load_stats_refresh_interval_in_seconds': 1
```

Comment:
You could possibly move it to cmdline above. We already use it to pass other tablet related options. Similarly for other tests.